File: trunk/download/stream/src/test/java/org/simpleframework/xml/transform/io/FileTransformTest.java
package org.simpleframework.xml.transform.io;

import java.io.File;
import junit.framework.TestCase;

public class FileTransformTest extends TestCase {
   
   public void testFile() throws Exception {
      File file = new File("..");
      FileTransform format = new FileTransform();
      String value = format.write(file);
      File copy = format.read(value);
      
      assertEquals(file, copy);      
   }
}

File: trunk/download/stream/port/src/main/Xml/Transform/FileTransformTest.cs
#region Using directives
using SimpleFramework.Xml.Transform;
using SimpleFramework.Xml;
using System;
#endregion
package SimpleFramework.Xml.transform;
public class FileTransformTest : TestCase {
   public void TestFile() {
      File file = new File("..");
      FileTransform format = new FileTransform();
      String value = format.write(file);
      File copy = format.read(value);
      AssertEquals(file, copy);
   }
}
}
